iceoryx_posh/include/iceoryx_posh/internal/popo/ports/base_port_data.hpp

```diff
@@ -49,7 +49,7 @@ struct BasePortData
     BasePortData& operator=(BasePortData&&) = delete;
     ~BasePortData() = default;
 
-    capro::ServiceDescription m_serviceDescription;
+    capro::ServiceDescription m_serviceDescription{capro::InvalidString, capro::InvalidString, capro::InvalidString};
```

Comment:
Please initialize the `m_serviceString` etc members in the `service_description.hpp ` with this values instead of doing it everywhere it is used

Reply:
Done